modules/assim.sequential/R/sda.enkf.R

```diff
@@ -1,27 +1,36 @@
 ##' @title sda.enkf
 ##' @name  sda.enkf
-##' @author Michael Dietze \email{dietze@@bu.edu}
+##' @author Michael Dietze and Ann Raiho \email{dietze@@bu.edu}
 ##' 
 ##' @param settings    PEcAn settings object
 ##' @param IC          data.frame of initial condition sample (nens X nstate)
-##' @param prior       data.frame of model parameter sample (nense X nstate)
-##' @param obs         data.frame of observations with columns: mean, sd
+##' @param prior       data.frame of model parameter sample (nens X nstate)
+##' @param obs.mean    data.frame of observations of the mean of variables
+##' @param obs.sd      data.frame of observations of the sd of variables
+##' @param processvar  flag for if process variance should be estimated or not
 ##' 
 ##' @description State Variable Data Assimilation: Ensemble Kalman Filter
 ##' 
 ##' @return NONE
 ##' 
-sda.enkf <- function(settings,IC,prior,obs){
+sda.enkf <- function(settings,IC,prior,obs.mean,obs.sd,processvar=NULL){
+  
+  if(is.null(processvar)) processvar = TRUE #tag for adding process variance estimation
```

Comment:
It appears that you're setting the default for processvar to NULL and then interpreting NULL as TRUE, which is going to be unintuitive to the external user. You should set the default value for processvar to what you think the default should be, not to NULL (personally I think FALSE should be the default, since that's the standard EnKF)